ansible/playbooks/rook.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: all
```

Comment:
Do we need to gather facts from all hosts (including e.g. `postgresql` and `kafka`)?

The next play will gather facts second time for `kubernetes_master`.

We do not use caching facts yet, there is issue #810 to implement.

Reply:
We don't need. Probably only repository is needed there.

Reply:
Done. :)